src/article/components/BoxComponent.js

```diff
@@ -2,25 +2,25 @@ import { NodeComponent } from '../../kit';
 
 export default class BoxComponent extends NodeComponent {
   render($$) {
-    let node = this.props.node;
+    const node = this.props.node;
 
     const Button = this.getComponent('button');
     const SectionLabel = this.getComponent('section-label');
 
     // Card
-    let el = $$('div')
+    const el = $$('div')
       .addClass('sc-box')
       .attr('data-id', node.id);
 
-    let titleEl = $$('div').addClass('sc-title');
+    const titleEl = $$('div').addClass('sc-title');
 
     // Label
     titleEl.append($$(SectionLabel, { label: 'box-title-label' }));
 
     // Remove Button
     titleEl.append(
       $$(Button, {
-        icon: 'remove',
```

Comment:
Will this affect any other icons, e.g. in metadata editor?

Reply:
I searched and replaced the references. It's a hangover of some work I did before, where I had created an entry for an icon that already had an entry, but under a different name. I've cleaned that up now to always use the 'trash' icon, and removed all the references that I could find to the 'remove' icon.
I've tested the UI, and all components that I changed and all 'seems' ok to me.